sdk/go/common/apitype/deployments.go

```diff
@@ -258,25 +304,31 @@ type DeploymentLogs struct {
 
 // A SecretValue describes a possibly-secret value.
 type SecretValue struct {
-	Value  string // Plaintext if Secret is false; ciphertext otherwise.
-	Secret bool
+	Value      string // Plaintext if Secret is false; ciphertext otherwise.
+	Ciphertext string // ciphertext
```

Comment:
Does this mean that the ciphertext gets duplicated now between `Value` and `Ciphertext`? I'm not sure we want to change this API.

Reply:
wops the comment is outdated, will change